akka-cluster/src/main/resources/reference.conf

```diff
@@ -40,21 +40,25 @@ akka {
     # Using auto-down implies that two separate clusters will automatically be
     # formed in case of network partition.
     #
-    # Don't enable this in production, see 'Auto-downing (DO NOT USE)' section 
+    # Don't enable this in production, see 'Auto-downing (DO NOT USE)' section
     # of Akka Cluster documentation.
     #
     # Disable with "off" or specify a duration to enable auto-down.
     # If a downing-provider-class is configured this setting is ignored.
     auto-down-unreachable-after = off
-    
+
+    # DEPRECATED: Specific to akka.cluster.AutoDown (which is discouraged to be used in production)
```

Comment:
Could we add a support in the typesafe-config?and make it logging the `deprecation` warning when parsing?

Reply:
Maybe, though only in application.conf then, and definitely not in reference.conf since I want to set the deprecated ones here after all for compatibility; Feel free to open a ticket there